src/main/java/org/opensearch/security/dlic/rest/support/Utils.java

```diff
@@ -213,4 +221,29 @@ public static String hash(final char[] clearTextPassword) {
                 .map(field -> prefix + field)
                 .collect(ImmutableSet.toImmutableSet());
     }
+
+    /**
+     * Add prefixes(_opendistro... and _plugins...) to rest API routes
+     * @param routes routes
+     * @return new list of API routes prefixed with _opendistro... and _plugins...
+     *Total number of routes is expanded as twice as the number of routes passed in
+     */
+    public static List<Route> addRoutesPrefix(List<Route> routes){
+        return addRoutesPrefix(routes, "/_opendistro/_security/api", "/_plugins/_security/api");
```

Comment:
Looks like we are adding all the routes to 
```
@Override    public List&lt;Route&gt; routes() {        return routes;    }
```
This will support both the RestAPIs but the older ones are not deprecated. Instead in our design we recommend a cleaner way to replace an existing route with a new route and mark the old one deprecated. OpenSearch automatically notifies a the user with a warning message when the older route is used.
You could use

```
@Override
public List<ReplacedRoute> replacedRoutes() {
 return ImmutableList
        .of(
            // delete anomaly detector document
            new ReplacedRoute(
                RestRequest.Method.DELETE,
                newRoute,
                RestRequest.Method.DELETE,
                oldRoute,
            )
        );
}
```
Please refer to the design for more details: https://github.com/opensearch-project/opensearch-plugins/blob/main/UPGRADING.md#rest-apis

Reply:
Thanks Sarat, as per our offline discussion, I'll open another PR for this right after the rc-1